contracts/src/v0.8/l2ep/dev/optimism/OptimismCrossDomainGovernor.sol

```diff
@@ -1,10 +1,14 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import "../interfaces/DelegateForwarderInterface.sol";
-import "../../../vendor/@eth-optimism/contracts/v0.4.7/contracts/optimistic-ethereum/iOVM/bridge/messaging/iOVM_CrossDomainMessenger.sol";
-import "../../../vendor/openzeppelin-solidity/v4.7.3/contracts/utils/Address.sol";
-import "./OptimismCrossDomainForwarder.sol";
+import {DelegateForwarderInterface} from "../interfaces/DelegateForwarderInterface.sol";
+// solhint-disable-next-line no-unused-import
```

Comment:
Do we still need the Forwarder Interface if we are not using it?

Reply:
It is used in the natspec comment, which is kinda annoying and this was the easiest fix. The comment inherits the comment from this import